raccoongang/edx-platform
# feat(mobile_api): Add course access object to mobile course info API

## Description:
This PR is a follow-up to FC-0031 project for Mobile API updates, and extends `BlocksInfoInCourseView` view with `CoursewareAccess` data for a user in the form:
```
"coursewareAccess": {
    "hasUnmetPrerequisites": <bool>,
    "isTooEarly": <bool>,
    "isStaff": <bool>
}
```

## Supporting information:
This PR further extends the mobile `BlocksInfoInCourseView` that was introduced in the https://github.com/openedx/edx-platform/pull/33296

## Testing instructions:
Run a **GET** `/api/mobile/{api_version}/course_info/blocks/?course_id=<course_id>&username=<username>&return_type=dict`.

Asses the response, check that fields "coursewareAccess" and "certificate" are in the response body.
